contracts/src/lib.rs

```
@@ -314,6 +316,26 @@ impl PortfolioRebalancer {
+        window_seconds: u64,
+    ) -> Result<(), Error> {
+        volatility::configure_volatility_strategy(
+            &env,
+            portfolio_id,
+            enabled,
+            threshold_bps,
+            window_seconds,
+        )
+    }
+
+    pub fn check_volatility_rebalance_needed(env: Env, portfolio_id: u64) -> bool {
```

Clippy (1.96.0)

[error] contract function name is too long

(error)